backend/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.19 AS builder
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main ./cmd/app
```

Comment:
I don't think you need `-installsuffix cgo` when CGO_ENABLED=0

Reply:
Thank you, removed